RPA.MIT.Notification.Function/Notification.cs

```diff
@@ -49,8 +48,15 @@ public Notification(INotificationTable notificationTable,  INotifyService notify
                 string templateName = notificationMsgObj.Action;
                 string scheme = notificationMsgObj.Scheme;
                 var templateId = _configuration[$"templates{templateName}"];
-                var emailAddress = _configuration[$"schemas{scheme}"];
                 string id = notificationMsgObj.Id;
+                var emailAddress = _configuration[$"schemas{scheme}"];
```

Comment:
This should probably override the other way round. the explicit email passed in in emailRecipient probably needs to be used in stead of the one pointed to by Scheme if it's passed in. Ie emailRecipient mandatory to pass?